src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -60,8 +64,10 @@ export const selectMarketDataById = createCachedSelector(
 )((_state: ReduxState, assetId?: AssetId): AssetId => assetId ?? 'assetId')
 
 // assets we have loaded market data for
-export const selectCryptoMarketDataIdsSortedByMarketCap = (state: ReduxState) =>
-  state.marketData.crypto.ids
+export const selectCryptoMarketDataIdsSortedByMarketCap = createDeepEqualOutputSelector(
+  selectMarketData,
```

Comment:
Since this is now *akchually* sorted market data, do we wanna rename it to `selectSortedMarketDataByMarketCap`? The rationale being:

1. Will make it explicit we're getting sorted data as an output
2. `selectSomeNoun` carries the mental-model of doing a simple slice property access kind of thing with zero to no derivation

Reply:
did you mean we should rename `selectMarketData` on `L19`?

Reply:
Correct @0xdef1cafe !